envbuilder.go

```diff
@@ -1201,3 +1195,70 @@ type osfsWithChmod struct {
 func (fs *osfsWithChmod) Chmod(name string, mode os.FileMode) error {
 	return os.Chmod(name, mode)
 }
+
+func findDevcontainerJSON(options Options) (string, string, error) {
```

Comment:
this looks like a good candidate for a direct unit test, if it does not already exist.

Reply:
fixed 👍